akka-diagnostics/src/main/resources/reference.conf

```diff
@@ -9,6 +9,230 @@
+  # the path of the configuration property to this list. When confirmed it
+  # will not warn about that setting any more.
+  confirmed-power-user-settings = []
+
+  # Application specific properties that are not defined by Akka should
+  # be placed outside the "akka" config tree, but if you still have to
+  # define such a configuration property inside "akka" you can confirm
+  # that it is not a typo or misplacement by adding the configuration
+  # path to this list. All properties starting with that path will not
+  # be checked for typos, i.e. you can add the path of a whole section
+  # here to skip everything inside that section.
+  confirmed-typos = []
+}
+#//#config-checker
+
+# Internal settings for the ConfigChecker. Don't change.
+akka.diagnostics.checker {
+
+  # Definition of all properties or sections that are classified as
+  # advanced "power user" settings. You should use 'confirmed-power-user-settings'
+  # instead of modifying this list.
+  power-user-settings = [
+    # akka-actor
+    akka.version,
+    akka.daemonic,
+    akka.actor.unstarted-push-timeout,
+    "akka.actor.router.type-mapping.*",
+    # task-peeking-mode is a power user setting but changed by Play reference-overrides
+    #akka.actor.default-dispatcher.fork-join-executor.task-peeking-mode,
+    akka.actor.default-dispatcher.thread-pool-executor.task-queue-size,
+    akka.actor.default-dispatcher.thread-pool-executor.task-queue-type,
+    akka.actor.default-dispatcher.attempt-teamwork,
+    akka.actor.default-dispatcher.shutdown-timeout,
+    "akka.actor.mailbox.*",
+    akka.actor.serializers.java,
+    akka.actor.serializers.bytes,
+    "akka.scheduler.*",
+    "akka.io.*",
+    akka.diagnostics.checker.power-user-settings,
+    akka.diagnostics.checker.disabled-typo-sections,
+
+    # akka-remote
+    akka.remote.use-passive-connections,
+    akka.remote.backoff-interval,
+    akka.remote.command-ack-timeout,
+    akka.remote.untrusted-mode,
+    akka.remote.artery.untrusted-mode,
+    akka.remote.log-received-messages,
+    akka.remote.artery.log-received-messages,
+    akka.remote.log-sent-messages,
+    akka.remote.artery.log-sent-messages,
+    "akka.remote.transport-failure-detector.*",
+    akka.remote.watch-failure-detector.implementation-class,
+    # akka.remote.watch-failure-detector.heartbeat-interval, FIXME shouldn't this be added?
+    akka.remote.watch-failure-detector.threshold,
+    akka.remote.watch-failure-detector.max-sample-size,
+    akka.remote.watch-failure-detector.min-std-deviation,
+    akka.remote.watch-failure-detector.unreachable-nodes-reaper-interval,
+    akka.remote.watch-failure-detector.expected-response-after,
+    akka.remote.system-message-buffer-size,
+    akka.remote.system-message-ack-piggyback-timeout,
+    akka.remote.resend-interval,
+    akka.remote.resend-limit,
+    akka.remote.initial-system-message-delivery-timeout,
+    "akka.remote.adapters.*",
+#    akka.remote.netty.tcp.transport-class,
```

Comment:
These were renamed in Akka 2.6 to `akka.remote.classic`. `akka.remote.classic` still exists in Akka 2.7.0 so we should probably keep them but with the akka.remote.classic` names.

Reply:
changed